apps/modernization-ui/src/components/Table/Table.tsx

```diff
@@ -19,6 +20,7 @@ type OldSelectionHandler = (event: ChangeEvent<HTMLInputElement>, item: any) =>
 export type Cell = {
     id: string | number;
     title: ReactNode | ReactNode[] | string;
+    actions?: any;
```

Comment:
Instead of any, lets be more explicit or use union types in cases where the type can be more than one possibility

Reply:
Okay I will use the specific type